core/modules/user/user.module

```diff
@@ -759,6 +767,20 @@ function user_account_form(&$form, &$form_state) {
   $form['#validate'][] = 'user_account_form_validate';
   module_load_include('password.inc', 'user', 'user');
   $reject_weak = user_password_reject_weak($user->name);
+
+  // Label the threshold for the password description.
+  $strength_threshold = $site_config->get('user_password_strength_threshold');
+  $threshold_label = t('excellent');
+  if ($strength_threshold < USER_PASSWORD_STRENGTH_FAIR) {
+    $threshold_label = t('weak');
+  }
+  if ($strength_threshold < USER_PASSWORD_STRENGTH_GOOD) {
```

Comment:
Is there a special reason, why you do three `if` in series for the same variable? Maybe I'm picky, but... `elseif` ...?

Reply:
I decided that the else isn't needed and this is slightly simpler. I think of it like stairs. If it doesn't stop at the first it'll just keep rolling down.